examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -504,18 +504,12 @@ impl FrontendNode {
             let data = vec![0x01, 0x02, 0x03];
             expect_eq!(
                 OakStatus::OK.value() as u32,
-                oak::wasm::channel_write(
-                    out_channel,
-                    data.as_ptr(),
-                    data.len(),
-                    &[] as *const u8,
-                    0,
-                )
+                oak_abi::channel_write(out_channel, data.as_ptr(), data.len(), &[] as *const u8, 0,)
```

Comment:
nit: does `rustfmt` leave the trailing comma?

Reply:
Somewhat surprisingly it does, but its ugly. Fixed